### ISSUE TITLE
PT translation: inexistent and polemic words

### ISSUE THREAD

**Issue by Massolari** [lipu-sona-pona/lipu-sona-pona.github.io#10]
While reading the course in Portuguese I realized that there are a few words that don't exist in Portuguese, for example:
- elu
- delu
- amigue
- mes
- filhes

Also, those words are the subject of much discussion in some groups, usually, related to politics and many people don't even know those words.
Some people say that this is necessary for non-binary people, but this is not a consensus even among non-binary people. Some want to use them, others say that they're wrong and shouldn't be used.

In conclusion, because we have so many discussions around those words and, in Brazil, it's not officially accepted (check https://www.in.gov.br/en/web/dou/-/portaria-n-604-de-27-de-outubro-de-2021-355520548), I suggest removing them from the course so it doesn't get involved in politics related discussions and bring the discussion to an environment not related to that subject.

I can make the PR for those changes if you would like.

**Comment by tbodt** [comment]
cc the portuguese translator @aliceofmeiji

**Comment by Daenyth** [comment]
> so it doesn't get involved in politics related discussions

I want to point out a misconception here. Removing queer-related language is not avoiding politics, it is itself a political stance.

I'm not a portuguese speaker, but I will consult with others. Bluntly though, I would probably not remove such wording, and certainly not with a brazilian government source as the justification, given the fascist leanings there.

**Comment by lipamanka** [comment]
hello! I am a nonbinary portuguese speaker. the use of elu, delu, amigue, etc. is very gender affirming for me and I don't see a problem with it! in portuguese contexts my friends use elu/delu pronouns for me, including in a college setting.

re: "officially accepted," this is not something linguists are concerned about when it comes to language. we do not care at all if any socially stratified group prescribes usage in a specific way. in fact, the use of gender neutral language in this translation of the portuguese lessons is a radical pro-trans choice that the translator made, and it coincides with the values of the toki pona community. 

if someone takes issue with the inclusion of this language used in this translation on the basis of not "agreeing with the transgender movement," we do not want them in our spaces. Hope this helps.

**Comment by jnpoJuwan** [comment]
hii, I'm another of the Portuguese speakers called. I understand what you're trying to get at, but it seems that the usage of elu/delu is in joint terms with other pronouns, so understandability would not be an issue.

on ma pona, some members decided that a good solution would be to add a notice, in short saying this. the text below is taken from jan Mimoku [lapingvino]:
> nesse curso usamos linguagem inclusiva. isso corresponde à cultura da comunidade. na prática isso significa o uso de -e em vez de -o/-a para falar de pessoas de forma neutra de gênero, e o uso de elu/delu como pronome neutro."

I myself don't use elu/delu pronouns or have met people who has, so I don't have much to say about how much it is used irl. but I will agree with those who say that it should be kept for the reasons stated above

**Comment by lipamanka** [comment]
Juwan I use elu/delu pronouns we're literally best friends wtffffff

**Comment by Massolari** [comment]
>  Clearly they exist and clearly some people use them, by your own words.

They don't exist **officially* in the language, as I posted, there is even a law forbidding its use.

>  Removing queer-related language is not avoiding politics
>  given the fascist leanings there.

Yes, we're leaning into politics discussions.

> if someone takes issue with the inclusion of this language used in this translation on the basis of not "agreeing with the transgender movement," we do not want them in our spaces. Hope this helps.

The point of the discussion is not about the use of "elu/delu" in Portuguese or "agreeing with the transgender movement"
The point is that those words are polemic and lead to politics discussions in Brazil, so IMO they shouldn't be in a course about teaching Toki Pona, it has its official ways to deal with non-binary pronouns and that's great, unfortunately, in Portuguese, we don't have such a way **officially**.

**Comment by Daenyth** [comment]
> there is even a law forbidding its use.

Yes, the brazilian government is fascist. Censoring language is a fascist tactic and we will not engage in it.

**Comment by Daenyth** [comment]
To expand a bit, the idea of words in a language being "official" or not is inherently political, 100%. It has **no basis in linguistic study at all**. It's not how human language works. Humans use language to communicate ideas, and "official" designation is literally just saying "you are only allowed to communicate in this way, and not in this other way".

**Comment by lipamanka** [comment]
I am currently in this very moment speaking with a sociolinguist who confirms that officiality is not linguistically meaningful. It is socially meaningful, as in it's used in social stratification and power structures, but linguistically literally nobody cares and I am getting quite frustrated.

**Comment by tbodt** [comment]
> The point of the discussion is not about the use of "elu/delu" in Portuguese or "agreeing with the transgender movement"
> The point is that those words are polemic and lead to politics discussions in Brazil

I may be mistaken, but my impression is these politics discussions are ultimately about "the transgender movement" so these wouldn't be separate concerns.
